README.md

```diff
@@ -5,29 +5,33 @@
 
 The Rust-ML working group is currently at an early stage of development, and is still deciding on several choices at an organizational level. This document will be updated accordingly as those decisions are made.
 
-### How can you contribute?
+## What do we do?
 
-Come say hello, read about, and weigh in on our [discussions](https://github.com/rust-ml/discussion)! We currently have several ongoing threads regarding upcoming scope of work, organizational structures, and more.
+In general, the Rust-ML working group's goals are to:
 
-The Rust-ML group has a Zulip channel. Feel free to take a look and say hi!
+- Create spaces where people interested in doing machine learning in Rust can meet, ask questions, and discuss their work
```

Comment:
Perhaps also:
 > Communicate interest in language features, like [GAT](https://github.com/rust-lang/rust/issues/44265), [const generics](https://github.com/rust-lang/rust/issues/44580), f16 types, which are important for easy-to-use machine learning libraries to the language team

Reply:
I'm a little hesitant to add this in unless there has been some direct contributions/advocacy for these languages features by members of the working group. Do you freely differently?

Reply:
no, there is no example of direct contributions to the language (at least what I'm aware of)